packages/core-modules/contracts/modules/ElectionModule.sol

```diff
@@ -48,7 +46,7 @@ contract ElectionModule is
 
         _createNewEpoch();
 
-        EpochData storage firstEpoch = _getEpochAtPosition(0);
+        EpochData storage firstEpoch = _getEpochAtIndex(0);
```

Comment:
Since the first epoch is now at index zero, I thought it'd be better to name these with "index" instead of "position".